.travis.yml

```diff
@@ -1,10 +1,20 @@
 language: php
+
 php:
-  - 5.5
-  - 5.4
+    - 5.4
+    - 5.5
+    - 5.6
+    - 7.0
+    - hhvm
+
 before_script:
-  - wget http://getcomposer.org/composer.phar
-  - php composer.phar install --dev --prefer-dist
+  - sudo apt-get -qq update > /dev/null
+  - sudo apt-get -qq install graphviz > /dev/null
+  - phpenv rehash > /dev/null
+  - composer selfupdate --quiet
```

Comment:
Besides getting an updated composer (which is enough I think) it avoids an annoying warning.